src/towncrier/build.py

```diff
@@ -135,7 +132,9 @@ def __main(
                 project_name = ""
 
     if project_date is None:
-        project_date = _get_date().strip()
+        project_date = date.today()
+    else:
+        project_date = parse(project_date).date()
```

Comment:
Just a minor comment

Can we just support ISO date format and then use stdlib `datetime.strptime` to parse the date ?

Not a big deal... just a shame that we dont't have "smart" parse function in stdlib.

At the same time, I think that we should encourage standard date representation, and only support ISO format.
... and it this way, we don't need to add yet another dependency